components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/auth/service/AuthServiceConstants.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.application.authentication.framework.util.auth.service;
+
+/**
+ * Constants class for Auth Service.
+ */
+public class AuthServiceConstants {
+
+    /**
+     * Enum for flow status.
+     */
+    public enum FlowStatus {
+
+        SUCCESS_COMPLETED,
+        FAIL_COMPLETED,
+        INCOMPLETE
```

Comment:
don't we have constants for these already?

Reply:
we have a specific set of constants related to authenticators